data/Rimpyyadav.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "Rimpy Yadav",
+  "type": "personal",
+  "bio": "Finding my better place in tech as Newbie :) Aspiring Backend webdev , Love to do Shuffle "
```

Please add a **'Comma'** at the end of this line :+1: 
```suggestion
  "bio": "Finding my better place in tech as Newbie :) Aspiring Backend webdev , Love to do Shuffle ",
```